source/frontend/src/features/contacts/contact/create/Organization/CreateOrganizationForm.tsx

```diff
@@ -132,9 +131,6 @@ const CreateOrganizationComponent: React.FC<FormikProps<IEditableOrganizationFor
 
   return (
     <>
-      {/* Router-based confirmation popup when user tries to navigate away and form has unsaved changes */}
-      <UnsavedChangesPrompt />
```

Comment:
This is a tricky one, because our regular modal doesn't pop up when navigating away, only when the user clicks "cancel" can we confirm with Julian/Dev that what they really want is for that modal not to display?